acme/acme/client_test.py

```diff
@@ -40,6 +40,8 @@ def setUp(self):
                 'https://www.letsencrypt-demo.org/acme/revoke-cert',
             messages.NewAuthorization:
                 'https://www.letsencrypt-demo.org/acme/new-authz',
+            messages.CertificateRequest:
```

Comment:
Was there a reason to add this as part of this PR? I don't see an explicit reference anywhere later on but I might be missing something.